bigchaindb_driver/driver.py

```diff
@@ -183,7 +181,11 @@ def transfer(self, transaction, *conditions, signing_key=None):
         signing_key = signing_key if signing_key else self.signing_key
         if not signing_key:
             raise InvalidSigningKey
-        transfer_transaction = transaction.transfer(list(conditions))
+        inputs = transaction.to_inputs()
```

Comment:
I think `get_inputs()` is a better name than `to_inputs()` (I assume this just extracts the inputs out of the transaction?).